src/tools/schema2kotlin.ts

```diff
@@ -246,7 +246,8 @@ abstract class Abstract${particle.name} : ${this.opts.wasm ? 'WasmParticleImpl'
     for (const connection of particle.connections) {
       const handleName = connection.name;
       const entityType = SchemaNode.entityTypeForConnection(connection, nodes);
-      const handleInterfaceType = this.handleInterfaceType(connection, entityType);
+      const devFriendlyName = SchemaNode.devFriendlyEntityTypeForConnection(connection, nodes);
```

Comment:
the variable name could still be entityType I think, the fact that you want to use the devFriendlyName is encapsulated by the method name used.

Reply:
Done